change/@azure-msal-browser-b8da8826-00fd-454d-bf25-e92d112713ca.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "patch",
```

No public API change.